thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/controller.js

```diff
@@ -209,6 +210,38 @@ export default Controller.extend({
     }
   }),
 
+  /**
+   * Preps a mailto link containing the currently selected metric name
+   * @method graphMailtoLink
+   * @return {String} the URI-encoded mailto link
+   */
+  graphMailtoLink: computed(
+    'alertData',
+    function() {
+      const alertData = this.get('alertData');
+      const fullMetricName = `${alertData.collection}::${alertData.metric}`;
+      const recipient = 'ask_thirdeye@linkedin.com';
+      const subject = 'TE Self-Serve Alert Page: error loading metric and/or alert records';
+      const body = `TE Team, please look into a possible inconsistency issue with [ ${fullMetricName} ] in alert page for alert id ${alertData.id}
+                    Alert page: https://thirdeye.corp.linkedin.com/app/#/manage/alert/${alertData.id}`;
```

Comment:
Could this be pulled from some existing constant somewhere? The domain`https://thirdeye.corp.linkedin.com`.  So that we have a single source of truth. Better yet, maybe have it dynamically pulled from query url.

Reply:
Yes. I'll get the URL instead.